src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,41 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: {
+    id: number;
+    title: string;
+    completed: boolean;
+    userId: number | string;
+    user?: {
+      id: number,
+      email: string;
+      name: string;
+      username: string;
+    };
+  };
+}
+
+const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const {
+    id, title, completed, user,
+  } = todo;
+
+  return (
+    <article
+      key={id}
+      data-id={id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed === true,
```

Comment:
```suggestion
        'TodoInfo--completed': completed,
```

Reply:
https://github.com/mate-academy/react_add-todo-form/pull/2015/files#r1320728679
didn't fix it